frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -148,10 +152,25 @@ export const CommandBar = () => {
             link: page.path,
         }));
         setSearchedPages(mappedPages);
+
+        const noResultsFound =
+            query.length !== 0 &&
+            mappedProjects.length === 0 &&
+            mappedPages.length === 0 &&
+            searchedFlagCount === 0;
+        if (noResultsFound) {
+            trackEvent('command-bar', {
+                props: {
+                    eventType: 'no search results found',
+                    query: query,
+                },
+            });
+        }
```

Comment:
For the feedback component (what were you looking for/would you like to request a feature?), this looks like we could reuse(?) but rewritten as a useState?